test/fixtures/basic/pages/keepalive/composables.ts

```diff
@@ -0,0 +1,6 @@
+export function useLifecyleLogs (name: string) {
```

Comment:
we should probably move this out to `test/fixtures/basic/composables` rather than keep them here, as this will create a page route.